pkg/translator/prometheusremotewrite/helper.go

```diff
@@ -255,10 +255,8 @@ func isValidAggregationTemporality(metric pmetric.Metric) bool {
 
 // addSingleHistogramDataPoint converts pt to 2 + min(len(ExplicitBounds), len(BucketCount)) + 1 samples. It
 // ignore extra buckets if len(ExplicitBounds) > len(BucketCounts)
-func addSingleHistogramDataPoint(pt pmetric.HistogramDataPoint, resource pcommon.Resource, metric pmetric.Metric, settings Settings, tsMap map[string]*prompb.TimeSeries) {
+func addSingleHistogramDataPoint(pt pmetric.HistogramDataPoint, resource pcommon.Resource, metric pmetric.Metric, settings Settings, tsMap map[string]*prompb.TimeSeries, baseName string) {
 	timestamp := convertTimeStamp(pt.Timestamp())
-	// sum, count, and buckets of the histogram should append suffix to baseName
-	baseName := prometheustranslator.BuildCompliantName(metric, settings.Namespace, settings.AddMetricSuffixes)
```

Comment:
Is it fine if the baseName given is just `""`? I'm expecting that is valid

Reply:
@jesusvazquez this PR doesn't introduce any change in this regard, does it? Or are you thinking whether this function should check if `baseName` is empty?

Reply:
Yes, that was my concern.

Reply:
It would be a programming error to send an empty `baseName`, but do you think I should add an explicit check that it's non-empty?